### PR TITLE
Document custom queryset required approach

### PR DESCRIPTION
**Pull request by nimasmi**
This may be OTT, but I ran into another example of where something that works in Django doesn't work quite the same way in Wagtail. Here's some documentation.

Ehkä tämä on liiallinen, vaan löysin jotakin jossa sopii Djangoille muttä ei sopi Wagtailille. Tässä on dokumentointia.

### CODE REVIEW

File: docs/topics/creating_pages.rst [hunk 1]
```diff
@@ -98,3 +98,46 @@ This is because ``Page`` enforces ordering QuerySets by path. Instead you must a
...
+            return self.filter(
+                start_date__gte=timezone.localtime(timezone.now()).date()
+            )
+
+    class EventPage(Page):
+        start_date = models.DateField()
+
+        objects = EventPageQuerySet.as_manager()  # will not work
+
+To use a custom manager you must choose a different attribute name. Make sure to subclass ``wagtail.wagtailcor.models.PageManager``.
```
Comment (gasman) [inline docs/topics/creating_pages.rst]: typo - wagtailcore

Comment (kaedroho) [inline docs/topics/creating_pages.rst]: Missing an 'e' on ``wagtailcore``

**Comment by timheap** [comment]
We came across this issue as well, and solved it by overriding the overridden `objects` with our own `objects` in a custom page metaclass, which inherits from the `PageBase` metaclass. It is multiple levels of things overriding each other and clobbering past assignments, which makes it particularly fun...

Adding it here in case it helps anyone. Would this be worth documenting, or is this too many levels of WTF hackery to be officially supported?

```python
import six

from django.db import models
from wagtail.wagtailcore import models as wagtailmodels


class MyPageBase(wagtailmodels.PageBase):
    def __init__(cls, name, bases, dct):
        super(PageBase, cls).__init__(name, bases, dct)

        # Clobber the forced attribute set in PageBase
        if 'template' not in dct:
            cls.template = "layouts/%s/%s.html" % (
                cls._meta.app_label, cls._meta.model_name)

        # Clobber the forced manager set in PageBase
        MyPageManager().contribute_to_class(cls, 'objects')


class MyPage(six.with_metaclass(MyPageBase, wagtailmodels.Page)):
    # Disable the creation of the related field accessor on Page
    # This prevents conflicts with model names and field names
    page_ptr = models.OneToOneField(wagtailmodels.Page, parent_link=True,
                                    related_name='+')

    # Set this here so autocompletion works. It will be clobbered by PageBase,
    # and then reset again by MyPageBase
    objects = PageManager()

    class Meta:
        abstract = True
```

**Comment by timheap** [comment]
See the above PR for a fix for this. Now you should be able to define custom Managers in the normal Django way.

**Pull request merged**